src/helm/benchmark/scenarios/audio_language/ultra_suite_asr_classification_scenario.py

```diff
@@ -69,33 +39,38 @@ def get_instances(self, output_path: str) -> List[Instance]:
         - Audio files (e.g., .mp3)
         - A JSON file with annotations containing 'answer' field
         """
-        print("Downloading SAA-Lab/SLPHelmManualLabels dataset...")
-        data_path = snapshot_download(
-            repo_id="SAA-Lab/SLPHelmManualLabels",
-            repo_type="dataset",
-            revision="38c2d7dab831acf8ccff0ca6f6463d6a8a0184ed",
-        )
+
+        audio_save_dir = os.path.join(output_path, "audio_files")
+        os.makedirs(audio_save_dir, exist_ok=True)
+        
+        print("Downloading SAA-Lab/SLPHelmUltraSuitePlus dataset...")
+        dataset = load_dataset("SAA-Lab/SLPHelmUltraSuitePlus") 
 
         instances: List[Instance] = []
         split: str = TEST_SPLIT
 
         # Find all pairs of audio and JSON files
         pairs = find_audio_json_pairs(data_path)
 
-        for audio_path, json_path in tqdm(pairs):
+        for row in tqdm(dataset["train"]):
+
+            label = row["disorder_class"]
 
-            # Load the annotation
-            with open(json_path, "r") as f:
-                annotation = json.load(f)
+            audio_path = row["audio"]
+            unique_id = str(uuid.uuid4())[:8] 
```

Comment:
This causes new UUIDs to be generated each time the scenario is run, which would cause many duplicates of the same data. It would be better to use the row index in the dataset as the unique ID. Even better would be to use the instance ID from the source dataset, if one exists.